api/src/domain/updateSidebarCards.test.ts

```diff
@@ -173,17 +173,20 @@ describe("updateRoadmapSidebarCards", () => {
   });
 
   describe("go-to-profile nudge", () => {
-    const poppyPhases: OperatingPhaseId[] = [
+    const doNotShowGoToProfileNudgePhases: OperatingPhaseId[] = [
       "GUEST_MODE",
       "NEEDS_TO_FORM",
       "NEEDS_TO_REGISTER_FOR_TAXES",
       "FORMED_AND_REGISTERED",
-      "UP_AND_RUNNING",
     ];
 
-    const oscarPhases: OperatingPhaseId[] = ["GUEST_MODE_OWNING", "UP_AND_RUNNING_OWNING"];
+    const showGoToProfileNudgePhases: OperatingPhaseId[] = [
+      "GUEST_MODE_OWNING",
+      "UP_AND_RUNNING_OWNING",
+      "UP_AND_RUNNING",
+    ];
```

Comment:
[pebble] Now that this logic is stored in operatingPhases, we can use a filter to determine these 2 lists instead of redeclaring them explicitly. For example, check out lines 273-278 of `profile-shared.test` to see what I mean.